docs/index.rst

```diff
@@ -40,6 +40,12 @@ P4 Control Plane User Guide
    clients/p4rt-ctl
    clients/sgnmi_cli
 
+.. toctree::
+   :maxdepth: 2
+   :caption: Features
+
+   guides/features/ipsec-offload
```

Comment:
```suggestion
   features/ipsec-offload
```
This is going to be a major division of the eventual document (as is Clients), so we might as well recognize this from the get-go.

I think the Guides section is going to morph into a different structure, probably broken into Building and Using divisions. The only thing left that might be called Guides is the three documents in the `guides/setup` directory.

Reply:
OK. Moved it to `docs/features` level and updated it